codebase/che/client.go

```diff
@@ -165,6 +165,61 @@ func (cs *StarterClient) CreateWorkspace(ctx context.Context, workspace Workspac
 	return &workspaceResp, nil
 }
 
+// StartExistingWorkspace starts an existing Che Workspace based on a repository
+func (cs *StarterClient) StartExistingWorkspace(ctx context.Context, workspaceName string) (*WorkspaceLink, error) {
+	log.Debug(ctx, map[string]interface{}{
+		"workspace_id": workspaceName,
+	}, "starting an existing workspace")
+
+	req, err := http.NewRequest("PATCH", cs.targetURL(fmt.Sprintf("workspace/%s", workspaceName)), nil)
```

Comment:
correct, it is implemented as part of https://github.com/redhat-developer/che-starter/pull/136
'GET workspace/' response which contains `ide start url` link